includes/widgets/google-maps.php

```diff
@@ -112,10 +112,12 @@ protected function register_controls() {
 					'api_key_notification',
 					[
 						'type' => Controls_Manager::RAW_HTML,
+						/* translators: %1$s: Integration settings link open tag, %2$s: Create API key link open tag, %3$s: Link close tag. */
 						'raw' => sprintf(
-							esc_html__( 'Set your Google Maps API Key in Elementor\'s <a href="%1$s" target="_blank">Integrations Settings</a> page. Create your key <a href="%2$s" target="_blank">here.', 'elementor' ),
-							Settings::get_url() . '#tab-integrations',
-							'https://developers.google.com/maps/documentation/embed/get-api-key'
+							esc_html__( 'Set your Google Maps API Key in Elementor\'s %1$sIntegrations Settings%3$s page. Create your key %2$shere.%3$s', 'elementor' ),
```

Comment:
Please use `1:` & `2:` in the translation comment.
```suggestion
							/* translators: 1: Integration settings link open tag, 2: Create API key link open tag, %3$s: Link close tag. */
							esc_html__( 'Set your Google Maps API Key in Elementor\'s %1$sIntegrations Settings%3$s page. Create your key %2$shere.%3$s', 'elementor' ),
```